std_msgs/String sign_name
geometry_msgs/PoseStamped sign_pose
